src/ecdsa/numbertheory.py

```diff
@@ -12,8 +12,7 @@
 from __future__ import division
 
 import sys
-from six import integer_types, PY2
```

Comment:
This breaks the `if GMPY2 or GMPY` case: it tries to do `integer_types = tuple(integer_types + (type(mpz(1)),))` and gets a `NameError` as a result.  I suggest adding `integer_types = int,` somewhere up here and reverting the other removals of `integer_types` in this file, as I just did in https://salsa.debian.org/python-team/packages/python-ecdsa/-/commit/3a95d8623d28d73c8c7877e6c3ddd5ee43e33ca4.